### PR TITLE
fix(rxMultiSelect): Fix overlapping rxMultiSelect issue

### PR DESCRIPTION
**Pull request by parlarjb**
https://jira.rax.io/browse/FRMW-641

This is one possible solution, James and Steven will be doing
PRs with other solutions

### CODE REVIEW

File: src/components/rxMultiSelect/scripts/rxMultiSelect.js [hunk 1]
```diff
@@ -111,6 +114,12 @@ angular.module('encore.ui.rxMultiSelect')
             scope.toggleDisplay = function (event) {
                 if (event.target === previewElement) {
                     scope.listDisplayed = !scope.listDisplayed;
+                    if (scope.listDisplayed) {
+                        // Ensure the dropdown from this rxMultiSelect doesn't
+                        // have weird issues with another rxMultiSelect that lives
+                        // right under it
+                        parentElement.css('z-index', 1000);
```
Comment (Droogans) [inline src/components/rxMultiSelect/scripts/rxMultiSelect.js]: What do you think about using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER instead?

You could subtract a few from it if you wanted to be extra safe for users who need to force something over this directive.

**Comment by CITguy** [comment]
Does it make sense to fix a CSS issue with javascript?

**Comment by parlarjb** [comment]
@CITguy That was a big part of the conversation that I had with @ssalinas24 and @100stacks.

If we can do it pure CSS, without major refactoring, then let's do that. Otherwise we have this as a solution.

**Comment by 100stacks** [comment]
@parlarjb, just pushed an alternate solution with PR https://github.com/rackerlabs/encore-ui/pull/1586.  Paired up with @CITguy on this one where we removed `rxSelect` dependencies to make `rxMultiSelect` a stand-alone module.

**Comment by CITguy** [comment]
Closing. Handled in #1586

**Pull request closed without merging**